R/profvis.R

```diff
@@ -55,17 +55,20 @@ profvis <- function(expr = NULL, interval = 0.01, prof_output = NULL,
     # vector. Make sure it's a single string.
     expr_source <- paste(expr_source, collapse = "\n")
 
+    opt_extension <- getOption("profvis.prof_extension")
+    prof_extension <- if (!identical(opt_extension, NULL)) opt_extension else ".prof"
```

Comment:
Could you just use `prof_extension <- getOption("profvis.prof_extension", default = ".prof")`?